diff --git a/plugins/ithor_plugin/ithor_tasks.py b/plugins/ithor_plugin/ithor_tasks.py
index 1245ce0..f1c3663 100644
--- a/plugins/ithor_plugin/ithor_tasks.py
+++ b/plugins/ithor_plugin/ithor_tasks.py
@@ -340,7 +340,7 @@ class ObjectManipTask(Task[IThorArmEnvironment]):
         # object_in_hand = self.env._objects_in_hand
         
         object_in_hand = False
-        print(self.env._objects_in_hand)
+        # print(self.env._objects_in_hand)
         for o in self.env._objects_in_hand:
             if self.task_info["object_type"] in o:
                 object_in_hand = True
